maquetado/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <link rel="stylesheet" type="text/css" href="application.css" />
+  </head>
+  <body>
+    <div class="book-detail">
+      <p class="book-title">Titulo del Libro</p>
```

Comment:
Remember to change this. I would use an H2 tag here and the rest can be H3 tags. 

I wouldn't use H1 tag because that's always used for the main page title.